bundles/org.openhab.binding.flicbutton/src/main/java/org/openhab/binding/flicbutton/handler/FlicDaemonBridgeHandler.java

```diff
@@ -0,0 +1,150 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.flicbutton.handler;
+
+import java.io.IOException;
+import java.net.UnknownHostException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+
+import org.openhab.binding.flicbutton.FlicButtonBindingConstants;
+import org.openhab.binding.flicbutton.internal.discovery.FlicButtonDiscoveryService;
+import org.openhab.core.thing.Bridge;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusDetail;
+import org.openhab.core.thing.binding.BaseBridgeHandler;
+import org.openhab.core.types.Command;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.flic.fliclib.javaclient.FlicClient;
+
+/**
+ * The {@link FlicDaemonBridgeHandler} handles a running instance of the fliclib-linux-hci server (flicd).
+ *
+ * @author Patrick Fink - Initial contribution
+ */
+public class FlicDaemonBridgeHandler extends BaseBridgeHandler {
+    private static final Logger logger = LoggerFactory.getLogger(FlicDaemonBridgeHandler.class);
+    private static final long REINITIALIZE_DELAY_SECONDS = 10;
+    // Config parameters
+    private FlicDaemonBridgeConfiguration cfg;
+    // Services
+    private FlicButtonDiscoveryService buttonDiscoveryService;
+    private Future<?> flicClientFuture;
+    // For disposal
+    private Collection<Future<?>> startedTasks = new ArrayList<>(2);
+    private FlicClient flicClient;
+
+    public FlicDaemonBridgeHandler(Bridge bridge, FlicButtonDiscoveryService buttonDiscoveryService) {
+        super(bridge);
+        this.buttonDiscoveryService = buttonDiscoveryService;
+    }
+
+    public FlicClient getFlicClient() {
+        return flicClient;
+    }
+
+    @Override
+    public void initialize() {
+        logger.debug("Initialize Fliclib bridge");
```

Comment:
Can this message be replaced by using the debugger or by increasing the framework's log level? See point 4 https://www.openhab.org/docs/developer/guidelines.html#f-logging